server/parsec/components/postgresql/realm.py

```diff
@@ -718,14 +874,10 @@ async def unshare(
         # approach by considering certificates don't change often so it's no big deal to
         # have a much more coarse approach.
 
-        # TODO: implement this
-        # assert (
-        #     org.last_certificate_or_vlob_timestamp is not None
-        # )  # Bootstrap has created the first certif
-        # if org.last_certificate_or_vlob_timestamp >= certif.timestamp:
-        #     return RequireGreaterTimestamp(
-        #         strictly_greater_than=org.last_certificate_or_vlob_timestamp
-        #     )
+        if last_realm_certificate_timestamp >= certif.timestamp:
```

Comment:
IMO, the conditional is more readable if `certif.timestamp` (what's being checked here) is on the left side:  
```suggestion
        if certif.timestamp <= last_realm_certificate_timestamp:
```